### DISCUSSION TITLE
SQLite Support?

### DISCUSSION THREAD

**Discussion by mrkaluzny** [decaporg/decap-cms#7193] [Ideas]
Hey, maybe this question is out of place, but we've been using Decap CMS for a long time starting with Netlify CMS, using Markdown files as the storage of data. 

This of course causes issues with relationships between content items, as well as creating searches and additional APIs. 

I think Decap is perfect for most cases, but would it be possible to add SQLite support instead of markdown/yaml/json files?

**Comment by mmkal** [comment]
My team would also be interested in this but it takes a fair amount of careful design. For example, how do you represent object widgets? Do you have a a table per collection and object/list widgets are json columns (this would mean no foreign keys for relation widgets) or do you attempt to fully normalize and have a new table for every complex sub-field (this would be much harder to write and make future-proof).

Do you have a specific idea of how this would work? This isn't an objection, I'd be genuinely interested in figuring out the answers here.
